service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -44,9 +41,7 @@
     private ExecutorService executor;
     private IotHubConnectionString iotHubConnectionString;
 
-    @Getter
-    @Setter
-    private Proxy proxy;
```

Comment:
Note that this would be a breaking change, but this getter/setter hasn't been in a released jar file yet, so it is still safe to change